ppr-ui/src/components/tables/mhr/HomeSectionsTable.vue

```diff
@@ -222,5 +305,30 @@ td {
   .v-list-item .v-list-item__title, .v-list-item .v-list-item__subtitle {
     line-height: 1.6 !important;
   }
+
+  .mhr-correction {
+    table tbody {
+      counter-reset: rowNumber;
+    }
+
+    tbody tr:not(.deleted-section) {
+      counter-increment: rowNumber;
```

Comment:
@cameron-eyds I used css to display numbers in each section row, skipping the deleted sections. 
![Screenshot 2024-03-19 at 18 57 01](https://github.com/bcgov/ppr/assets/2333290/3e7b83e9-af18-4a4a-a6fe-3867ba077f2d)

Reply:
Just wanted to bring it up, in case there are some cases we might not want to use css to do it. But so far, it looks like it's only presentational.